scripts/pre_commit_linter.py

```diff
@@ -2745,11 +2634,13 @@ def perform_all_lint_checks(self):
 
         common_messages = super(
             JsTsLintChecksManager, self).perform_all_lint_checks()
+        arg1 = (self.js_filepaths, _STDOUT_LIST, self.process_manager)
+        arg2 = (self.all_filepaths, _STDOUT_LIST, self.process_manager)
```

Comment:
Can we add a proper name here for args*? (If these are not used in multiple places then you can use them directly instead of creating variable.)